packages/forms-web-app/src/controllers/selected-appeal/index.js

```diff
@@ -42,13 +46,18 @@ exports.get = (layoutTemplate = 'layouts/no-banner-link/main.njk') => {
 			throw new Error('no session email');
 		}
 
-		const user = await apiClient.getUserByEmailV2(userEmail);
+		const user = await req.appealsApiClient.getUserByEmailV2(userEmail);
 
-		const caseData = await apiClient.getUsersAppealCase({
-			caseReference: appealNumber,
-			role: userType,
-			userId: user.id
-		});
+		let [caseData, events] = await Promise.all([
```

Comment:
Does this need to be a let?

Reply:
yes probably should be, still half imagine const means immutable so probably just used let as there is a sort on event.
Am getting this in to allow other tickets to progress, feel free to change in your PR